janusgraph-dist/src/assembly/static/conf/gremlin-server/gremlin-server-berkeleyje-es.yaml

```diff
@@ -33,7 +33,7 @@ serializers:
     - { className: org.apache.tinkerpop.gremlin.driver.ser.GryoMessageSerializerV1d0, config: { ioRegistries: [org.janusgraph.graphdb.tinkerpop.JanusGraphIoRegistry] }}
     - { className: org.apache.tinkerpop.gremlin.driver.ser.GryoLiteMessageSerializerV1d0, config: {ioRegistries: [org.janusgraph.graphdb.tinkerpop.JanusGraphIoRegistry] }}
     - { className: org.apache.tinkerpop.gremlin.driver.ser.GryoMessageSerializerV1d0, config: { serializeResultToString: true }}
-    - { className: org.apache.tinkerpop.gremlin.driver.ser.GraphSONMessageSerializerGremlinV2d0, config: { ioRegistries: [org.janusgraph.graphdb.tinkerpop.JanusGraphIoRegistry] }}
```

Comment:
Wouldn't be the correct way to add both like GraphSONMessageSerializerGremlinV1d0 and GraphSONMessageSerializerV1d0?

Reply:
I'm not sure, because in GraphSONMessageSerializerGremlinV1d0  GraphSONMapper's typeInfo is TypeInfo.PARTIAL_TYPES,but in GraphSONMessageSerializerV1d0 the typeInfo is TypeInfo.NO_TYPES, i don’t know what’s the difference,may need to be fully tested,so i didn't modify it.

Reply:
You could check that you don't get the exception, if you just add the new line and no other changes.

Reply:
I have checked the problem, i also try test  change GraphSONMessageSerializerGremlinV1d0 to GraphSONMessageSerializerV1d0, but it didn't workers with v2、v3 because GraphSONMessageSerializerGremlinV1d0 couldn't accept  application/vnd.gremlin-v3.0+json ,it only accept application/json

Reply:
Sorry, my idea was to add both following instead of replacement.
```yaml
 - { className: org.apache.tinkerpop.gremlin.driver.ser.GraphSONMessageSerializerGremlinV2d0, config: { ioRegistries: [org.janusgraph.graphdb.tinkerpop.JanusGraphIoRegistry] }}
 - { className: org.apache.tinkerpop.gremlin.driver.ser.GraphSONMessageSerializerV2d0, config: { ioRegistries: [org.janusgraph.graphdb.tinkerpop.JanusGraphIoRegistry] }}
```

Reply:
could you tell me the reason？

Reply:
It seems so that you cannot configure both.